org.eclipse.xtext.xbase.testing/src/org/eclipse/xtext/xbase/testing/CompilationTestHelper.java

```diff
@@ -212,6 +226,16 @@ public void compile(Iterable<? extends CharSequence> sources, IAcceptor<Result>
 	public void compile(final ResourceSet resourceSet, IAcceptor<Result> acceptor) {
 		try {
 			List<Resource> resourcesToCheck = newArrayList(resourceSet.getResources());
+			if (generatorConfigProvider instanceof GeneratorConfigProvider) {
+				GeneratorConfigProvider configProvider = (GeneratorConfigProvider) generatorConfigProvider;
+				GeneratorConfig config = generatorConfigProvider.get(null);
```

Comment:
I'd prefer a constructor call to GeneratorConfig, instead of call the provider with null